ugs-core/src/com/willwinder/universalgcodesender/AbstractCommunicator.java

```diff
@@ -74,6 +74,8 @@ public AbstractCommunicator() {
     /*********************/
     abstract public void setSingleStepMode(boolean enable);
     abstract public boolean getSingleStepMode();
+    abstract public void setSingleBlockMode(boolean enable);
```

Comment:
I know there isn't much javadoc on the other methods. But I find it useful having javadocs on interfaces and abstract methods to get a hint of what the method does and its intended use.